components/raftstore/src/store/config.rs

```diff
@@ -278,6 +278,10 @@ pub struct Config {
 
     #[doc(hidden)]
     pub max_snapshot_file_raw_size: ReadableSize,
+
+    /// The capacity of in-memory pessimistic locks. Make it smaller than `raft-max-size-per-msg` so that
```

Comment:
Can we add a check for this in TiKV?

Reply:
Probably we have to spare some space for the size bloating. We'd better calculate the amplification ratio..